### ISSUE TITLE
RNN module weights are not part of single contiguous chunk of memory.

### ISSUE THREAD

**Issue by quenting44** [KongHag/emotion_project#1]
https://github.com/KongHag/emotion_project/blob/6a5a7c0737ea4a56dc7544ab253dc9efc1009c0e/training.py#L70

Quand on lance sur GPU on a :

`/opt/pytorch/1.0.0/venv/lib/python3.6/site-packages/torch/nn/modules/rnn.py:179: RuntimeWarning: RNN module weights are not part of single contiguous chunk of memory. This means they need to be compacted at every call, possibly greatly increasing memory usage. To compact weights again call flatten_parameters().
  self.dropout, self.training, self.bidirectional, self.batch_first)`

Il semblerait que ce soit un problème de version. (https://github.com/zihangdai/mos/issues/11#issuecomment-451757829)

**Comment by quenting44** [comment]
Résolu en ajoutant `model.to(device)`
